collections/src/test/kotlin/ktx/collections/MapsTest.kt

```diff
@@ -393,4 +393,82 @@ class MapsTest {
     assertEquals("Three", gdxArrayMap[3])
     assertEquals("Four", gdxArrayMap[4])
   }
+
+  @Test
+  fun `should return existing value for GdxMap when key exists`() {
+    val map = gdxMapOf("42" to 42)
+
+    val actual = map.getOrPut("42") { 43 }
+
+    assertEquals(42, actual)
```

Comment:
Ensure that the value was not replaced in the map, i.e. `"42"` should still point to `42`.

Reply:
Same goes for other similar tests.